drone_localization/streamlit/app.py

```
@@ -21,7 +21,9 @@
             img_b64 = base64.b64encode(img_bytes).decode("utf-8")
 
             # Запрос к API
-            res = requests.post(f"{API_URL}/api/localize", json={"drone_image": img_b64})
+            res = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)